docs/ilm-setup.asciidoc

```diff
@@ -0,0 +1,189 @@
+== Manual index lifecycle management
+
```

Comment:
How about an intro sentence about automatically cleaning / limiting unbounded storage consumption first?

Reply:
Do we need to point out that this requires xpack?